### PR TITLE
From the embedded page, link to the tools page to set up editors

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

File: getting-started/embedded-swift/index.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ Hooray! Our first *host-side* Embedded Swift program is working!
 - The [Embedded Swift Vision Document](https://github.com/swiftlang/swift-evolution/blob/main/visions/embedded-swift.md) will give you an overview of the approaches and goals of Embedded Swift, and also what exactly is in the Embedded Swift language subset.
 - The [collection of Embedded Swift example projects](https://github.com/apple/swift-embedded-examples) on GitHub shows on which embedded devices can Swift work today, and the examples can also be used as templates for your own projects.
 - The [Embedded Swift User Manual](https://github.com/apple/swift/blob/main/docs/EmbeddedSwift/UserManual.md) describes how to use the Embedded Swift compilation mode and how to interact with the compiler.
-- You might want to set up [Swift editor integration via SourceKit-LSP](https://github.com/swiftlang/sourcekit-lsp) to get features like indexing, autocomplete, jump-to-definition, show declarations/documentations, etc. to work in your favorite editor or IDE.
+- To set up Swift integration in your editor, read one of our [setup guides](https://www.swift.org/tools/).
```
Comment (dempseyatgithub) [inline getting-started/embedded-swift/index.md:218]:
```suggestion
- The [Tools page](https://www.swift.org/tools/) has guides for setting up Swift integration in your editor to enable features like indexing, autocomplete, jump-to-definition, and others.
```

All the other bullet points in the list start with the name of the resource being recommended. This suggestion matches the style of the other bullet points and preserves a shortened version of the benefit to the reader of following the link.

**Review by dempseyatgithub — COMMENTED** [review]
Thank you for the PR and for updating to a more appropriate link!

I didn't do a full on 'request change' but please consider using my suggested change.

File: getting-started/embedded-swift/index.md (hunk 1 above)
Comment (rauhul) [reply getting-started/embedded-swift/index.md:218]: Maybe also include the `#editors` anchor?

Comment (ahoppen) [reply getting-started/embedded-swift/index.md:218]: Good suggestion. Thanks!

**Comment by rauhul** [comment]
@swift-ci test

**Review by dempseyatgithub — APPROVED** [review]
This looks good - thank you!

**Pull request merged**